new-frontiers-14/frontier-station-14
# Add an item appraisal cartridge for PDAs

## About the PR
Adds an item appraisal cartridge for PDAs. Supports left-click interaction to appraise items while the program is active, and keeps a rolling history of those results visible in the PDA interface. Right-click appraisal utility verb is available for cases like bags or deep fryers that the PDA would be placed into on left-click, but those results won't be stored in history due to implementation horrors (see comments in code). Cribs heavily from the Netprobe cartridge for UI. 



## Why / Balance
Right now with very few exceptions a medical PDA is the obvious best choice for loadouts, and most of the alternatives have little to no additional functionality. More choices that do something beyond just having a different sprite is fun.


## How to test
Start up a local instance, spawn in an appraisal cartridge and whatever PDAs you think look coolest. Install the cartridge and exercise the program on the items in the sandbox zone, verify things function as expected. 


## Media
![appapp1](https://github.com/user-attachments/assets/46bc6630-1298-4033-b19d-13cf5eb0b2e8)
![appapp2](https://github.com/user-attachments/assets/238826dc-5c3a-4cd2-9fab-05db1428ec7f)



## Requirements


## Breaking changes


**Changelog**



:cl:
- add: Added an appraisal program cartridge for PDAs
